cli/tools/run/mod.rs

```diff
@@ -77,9 +77,14 @@ pub async fn run_script(
   let worker_factory = factory.create_cli_main_worker_factory().await?;
   let mut worker = worker_factory
     .create_main_worker(mode, main_module.clone())
-    .await?;
-
-  let exit_code = worker.run().await?;
+    .await
+    .inspect_err(|e| {
+      deno_runtime::ops::otel::report_event("deno_boot_failure", e)
+    })?;
+
+  let exit_code = worker.run().await.inspect_err(|e| {
+    deno_runtime::ops::otel::report_event("deno_uncaught_exception", e)
```

Comment:
I'm a bit worried about spread of these semi-random strings throughout the codebase. Maybe we could create a `deno_otel` crate that all other crates pull that contains a list of `static: &str EVENT_NAME = "event_name;` vars that we can use?